src/components/docs/OverviewReferences/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import CardsWrapper from '../Cards/CardsWrapper';
+import SimpleCard from '../Cards/SimpleCard';
+
+const OverviewReferences = ({ exclude }) => (
```

Comment:
This doesn't handle cases where they want to exclude multiple sections, or allow for different ordering. Also, adding a new section seems like extra work. This seems a little prescriptive.

 Can we just split these out so the usage is:

```
<OverviewWallet />
<OverviewCore />
```

Reply:
@jjwill  So you're saying instead of having one section called OverviewReferences, to split it out into one for each reference?